Makefile

```diff
@@ -86,6 +86,11 @@ testloop: receptor
 ci: pre-commit build-all test
 	@echo "All done"
 
+version:
+	@echo $(VERSION) > .VERSION
+	@echo $(VERSION) > receptorctl/.VERSION
```

Comment:
It should not be necessary to create a copy in the receptorctl directory, because receptorctl's setup.py also looks for `../.VERSION`.